code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -159,3 +159,22 @@
 	dismemberment = 0
 	weaken = 0
 	breakthings = FALSE
+
+//toy magazine
+/obj/item/ammo_box/magazine/toy/sniper_rounds
+	name = "donksoft Sniper magazine"
+	icon_state = ".50mag"
```

Comment:
Maybe a different sprite to show that its a toy? IRL nerf guns have neon orange mags, Im not saying that should be done here but it could be different

Reply:
See above